package.json

```diff
@@ -66,6 +66,8 @@
     "vr:test": "jest --config jest.visual-regression.config.js",
     "vr:test:ci": "yarn run vr:test --runInBand --coverage",
     "vr:test:ci:local": "yarn run vr:test --coverage",
+    "vr:test:regression": "yarn start-server-and-test vr:start http://localhost:4200 vr:percy:run",
+    "vr:percy:run": "export PERCY_TOKEN=fb4bc79a6546164e53d208391f04b7fd84fb0afb0673fa47d285b36565b61d8c && percy exec -- cypress run --spec 'cypress/integration/visual-regression/*.js'",
```

Comment:
This token should probably not be exposed to the world.. Is this here to allow local runs?